### PR TITLE
Issue/4326 plans no store

### LINKED ISSUE #4326: Update plans UI - no play store
It has been decided that integration with the iOS and Android app stores isn't practical, so we should remove (or at least disable) all of that code. Instead, the plans UI will simply contain a section letting the user know where to go to upgrade/manage their plans.

### PR DESCRIPTION
**Pull request by nbradbury**
Resolves #4326 - removes Google Play integration (ie: in-app billing) from Plans and adds a "Manage your plan at WordPress.com/plans" section to the plans UI.

**Note:** Right now plans are only exposed to devs/testers - end users won't see this feature yet.

android-plans

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java [hunk 1]
```diff
@@ -260,10 +261,10 @@ public void onClick(View v) {
     }
 
     /*
-     * plans is a work-in-progress and is currently only exposed to alpha testers
+     * plans is a work-in-progress and isn't exposed yet
      */
     private static boolean isPlansEnabled() {
-        return AppPrefs.isInAppBillingAvailable();
+        return PackageUtils.isDebugBuild();
```
Comment (maxme) [inline WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java]: IMO, that's a bit weird to remove the feature flag and enable the Plans in debug mode only. Can we keep the feature flag and eventually rename it to `PLANS_AVAILABLE`?

**Comment by maxme** [comment]
Translations of the Plan names come from wpandroid for tab titles and from the wpcom rest api for the fragment title, that could lead to discrepancies:

https://cloudup.com/clVnsYJ5NEG

Not sure how to solve this. That's a not a big deal, maybe just ping i18n.

**Comment by nbradbury** [comment]
> Translations of the Plan names come from wpandroid for tab titles and from the wpcom rest api for the fragment title

I'm not sure that's correct. I think the tab titles are being set [here](https://github.com/wordpress-mobile/WordPress-Android/blob/issue/4326-plans-no-store/WordPress/src/main/java/org/wordpress/android/ui/plans/adapters/PlansPagerAdapter.java#L43) based on the [product_name_short](https://github.com/wordpress-mobile/WordPress-Android/blob/issue/4326-plans-no-store/WordPress/src/main/java/org/wordpress/android/ui/plans/models/Plan.java#L78) field returned by the backend.

The title comes from [the full product name](https://github.com/wordpress-mobile/WordPress-Android/blob/issue/4326-plans-no-store/WordPress/src/main/java/org/wordpress/android/ui/plans/models/Plan.java#L65).

**Comment by maxme** [comment]
> I'm not sure that's correct. I think the tab titles are being set here based on the product_name_short field returned by the backend.

Right, I thought the first network request was made after tab loading.

**Comment by maxme** [comment]
LGTM except for one edge case, when a tab is not scrollable (Text fits the size of the screen perfectly). The bottom bar hides the end of the text and you have to "scroll" the unscrollabe view to make the bottom bar disappear, which is a bit weird. I'd go with the simplest solution here and have the bottom bar always visible and the scrollview in between to the tab bar and the bottom bar.

See: https://cloudup.com/cpqIoDXFgGq

**Comment by nbradbury** [comment]
> I'd go with the simplest solution here and have the bottom bar always visible and the scrollview in between to the tab bar and the bottom bar.

@maxme I took your advice in c8a3124

**Comment by maxme** [comment]
:shipit:

**Pull request merged**